src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs

```diff
@@ -42,7 +41,7 @@ public class Settings : ISettings
             new[] { "*.config" } :
             new[] { "*.Config", "*.config" };
 
-        private SettingsFile _settingsHead { get; }
+        private readonly SettingsFile _settingsHead;
```

Comment:
Changes like this are unnecessary with C# 6 and later. 

https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-6#read-only-auto-properties

Reply:
Auto properties generate both a property and a field.  We only need a field.